plugin/src/main/java/io/github/pronze/sba/game/InvisiblePlayerImpl.java

```diff
@@ -111,61 +113,63 @@ private void showArmor() {
                         convert(helmet),
                         convert(chestplate),
                         convert(leggings),
-                        convert(boots), convert(currentHand)).sendPacket(PlayerMapper.wrapPlayer(pl)));
+                        convert(boots), convert(currentHand)).sendPacket(Players.wrapPlayer(pl)));
     }
 
-    public Item convert(ItemStack itemStack) {
-        return ItemFactory.build(itemStack).orElse(ItemFactory.getAir());
+    public org.screamingsandals.lib.item.ItemStack convert(ItemStack itemStack) {
+        return Objects.requireNonNullElse(ItemStackFactory.build(itemStack), ItemStackFactory.getAir());
     }
 
     private void hideArmor() {
         var hiddenPlayerTeam = arena.getGame().getTeamOfPlayer(hiddenPlayer);
-        final var airStack = ItemFactory.getAir();
+        final var airStack = ItemStackFactory.getAir();
         arena.getGame()
                 .getConnectedPlayers()
                 .stream()
                 .filter(pl -> !hiddenPlayerTeam.getConnectedPlayers().contains(pl))
                 .forEach(pl -> {
                     getEquipPacket(airStack, airStack, airStack, airStack,
                             convert(hiddenPlayer.getInventory().getItemInMainHand()))
-                            .sendPacket(PlayerMapper.wrapPlayer(pl));
+                            .sendPacket(Players.wrapPlayer(pl));
                 });
     }
 
     public void refresh() {
         // TODO Auto-generated method stub
-        final var airStack = ItemFactory.getAir();
+        final var airStack = ItemStackFactory.getAir();
         var hiddenPlayerTeam = arena.getGame().getTeamOfPlayer(hiddenPlayer);
-        Tasker.build(() -> {
+        Tasker.run(DefaultThreads.GLOBAL_THREAD, () -> {
             arena.getGame()
                     .getConnectedPlayers()
                     .stream()
                     .filter(pl -> hiddenPlayerTeam == null || !hiddenPlayerTeam.getConnectedPlayers().contains(pl))
                     .forEach(pl -> {
                         getEquipPacket(airStack, airStack, airStack, airStack,
                                 convert(hiddenPlayer.getInventory().getItemInMainHand()))
-                                .sendPacket(PlayerMapper.wrapPlayer(pl));
+                                .sendPacket(Players.wrapPlayer(pl));
                     });
-        }).afterOneTick().start();
+        });
```

Comment:
This should be delayed by 1 tick, unless Tasker.run does that automatically, instead of 0 ticks

Reply:
Tasker.run executes BukkitScheduler.runTask and its javadoc states that the operation will run in the next tick. The alternative FoliaTasker implementation uses GlobalRegionScheduler.run which states the same.

I have checked what 2.0.2 did when afterOneTick has been applied. It used the same method
```java
if (builder.isAfterOneTick()) {
    return AbstractTaskerTask.of(builder.getTaskId(), scheduler.runTask(plugin, runnable), builder.getStopEvent());
}
```